helm/sealed-secrets/Chart.yaml

```diff
@@ -16,4 +16,4 @@ maintainers:
     email: mmikulicic@gmail.com
 name: sealed-secrets
 type: application
-version: 2.3.0
+version: 2.4.0
```

Comment:
Could you please remove this change? We are going to release a new version of the chart today (matching new release `v0.18.1`) so we will create a new PR increasing the version ourselves.